pkg/auth/authorizer/abac/abac.go

```diff
@@ -98,7 +99,15 @@ func NewFromFile(path string) (policyList, error) {
 }
 
 func (p policy) matches(a authorizer.Attributes) bool {
-	if p.subjectMatches(a) {
+
+	switch {
+	case p.NonResourcePath != "":
```

Comment:
not sure what the behavior should be if you set both p.NonResourcePath and p.Namespace/p.Resource... that seems like an error... I think want a single thing allowed per policy rule